src/charts/axis.js

```diff
@@ -161,10 +162,14 @@ export default function Axis (_container) {
   }
 
   function drawAxis () {
-    cache.root.select(".axis.x")
+    const xAxis = cache.root.select(".axis.x")
```

Comment:
Wouldn't be easier to simply hide the .domain for these chart types? It would also avoid adding chartType as a dependency for the axis component and styling logic in the render logic.

Reply:
where would the logic for hiding it live? how would that work? IMO it makes sense to do it here as it's where the axises are being constructed. 

I suppose somewhere in `bar.js` there could be some logic that selects the x axis's `path.domain` and hides / removes it. But it feels odd to have it there. 

Or in a SCSS partial for the bar component there could be some name-spacing that allows for hiding it, but that might get ugly with using lots of selectors.

IMO it seems more semantic to do it in the axis component as that's where the axises are being rendered.

Reply:
In css, it would be nothing more than:
```
.bar-type .domain {
    display: none;
}

Reply:
are you sure that would override D3's inline styles?